astro/src/content/docs/extend/examples/multi-application-dashboard.mdx

```diff
@@ -0,0 +1,166 @@
+
+This guide will demonstrate a simpler and safer way to authenticate users from a central dashboard by making each application link in the dashboard point to the application's login page. From that point onward, authentication follows the standard OAuth Authorization Code Flow.
+
+The next section of this guide will show you how to make a dashboard for two existing FusionAuth applications in the same tenant. All this requires is customizing the theme for the index landing page of the FusionAuth site. The guide will first show you how to install FusionAuth and make two simple web application web servers. If you already have a FusionAuth installation with existing applications, you can skip ahead to the section that creates the dashboard.
+
+If you want to follow along with the full guide, you will need Docker installed.
+
+## Download The Example Repository And Run FusionAuth
+
+Use `git clone` to clone the repository at https://github.com/fusionauth/fusionauth-example-multiapp-dashboard, or download and unzip it. 
+
+Open a terminal in the directory containing the repository files.
+
+Run the command below to start FusionAuth.
+
+  ```sh
+  docker compose up
+  ```
+
+<Aside type="note">
+  If you have completed any FusionAuth tutorials before, you might need to first delete any existing FusionAuth containers with the same name and the database volumes. Run the command below to do that.
+
+  ```sh
+  docker ps rm fa faDb; docker compose down -v
+  ```
+</Aside>
+
+Leave FusionAuth running. 
+
+In a new terminal, run the commands below to start a web server for the Changebank app, which uses FusionAuth for authentication. 
+
+  ```sh
+  cd bankApp
+  docker run --platform=linux/amd64 --rm -v ".:/app" -w "/app" node:23-alpine3.19 sh -c  "npm install"
+  docker run --platform=linux/amd64 --rm --network faNetwork -p 3000:3000 -v ".:/app" -w "/app" -e "PORT=3000" node:23-alpine3.19 sh -c  "npm run start"
+  ```
+
+The Changebank app is available at http://localhost:3000. 
+
+To learn how to make a simple Node.js app that uses FusionAuth, read the [quickstart](/docs/quickstarts/quickstart-javascript-express-web).
+
+In a third terminal, run the commands below to start a web server for a second app that uses FusionAuth for authentication. The app is called Changeinsurance, and is available at http://localhost:3001.
+
+  ```sh
+  cd insuranceApp
+  docker run --platform=linux/amd64 --rm -v ".:/app" -w "/app" node:23-alpine3.19 sh -c  "npm install"
+  docker run --platform=linux/amd64 --rm --network faNetwork -p 3001:3001 -v ".:/app" -w "/app" -e "PORT=3001" node:23-alpine3.19 sh -c  "npm run start"
+  ```
+
+Before making the dashboard, check that you can log in to all three applications. Either use an incognito browser window or don't enable [Keep me signed in](/docs/lifecycle/authenticate-users/logout-session-management#fusionauth-sso) when logging in, otherwise, you won't see the login form in the rest of this guide:
```

Comment:
```suggestion
Before making the dashboard, check that you can log in to all three applications. Either use an incognito browser window or don't enable <InlineUIElement>[Keep me signed in](/docs/lifecycle/authenticate-users/logout-session-management#fusionauth-sso)</InlineUIElement> when logging in, otherwise, you won't see the login form in the rest of this guide:
```